elections/uk/management/commands/uk_create_elections_from_every_election.py

```diff
@@ -0,0 +1,193 @@
+
+        organization_extra = self.get_or_create_organisation(
+            election_model, election_dict)
+
+        if election_dict['children']:
+            for division_id in election_dict['children']:
+                self.add_single_division(
+                    election_model, organization_extra, division_id)
+        else:
+            # This is an election to the org directly, with no divisions
+            # e.g. a mayor or PCC
+            self.add_area(
+                election_dict, election_model, organization_extra)
+
+    def get_election(self, election_dict):
+        election_id = election_dict['election_id']
+        election_date = election_dict['poll_open_date']
+
+        # TODO Set current from the API
+        current = True
+
+        return Election.objects.update_or_create(
+            slug=election_id,
+            election_date=election_date,
+            current=current,
+            defaults={
+                "name": election_dict['election_title'],
+                "for_post_role": election_dict['elected_role'],
+                "candidate_membership_role": "Candidate",
+                "show_official_documents": True,
+            }
+        )[0]
+
+    def add_single_division(self, election_model,
+                            organization_extra, division_id):
+        url = "{}api/elections/{}".format(
+            self.EE_BASE_URL, division_id)
+        area_info = requests.get(url).json()
+
+        self.add_area(area_info, election_model, organization_extra)
+
+    def get_or_create_organisation(self, election_model, election_dict):
+        org_name = election_dict['organisation']['official_name']
+        org_slug = election_dict['organisation']['slug']
+        classification = election_dict['organisation']['organisation_type']
+
+        try:
+            organization_extra = OrganizationExtra.objects.get(
+                slug=org_slug)
+            organization = organization_extra.base
+        except OrganizationExtra.DoesNotExist:
+            organization, _ = Organization.objects.get_or_create(
+                name=org_name,
+                defaults={'classification': classification})
+            organization_extra = OrganizationExtra.objects.create(
+                base=organization,
+                slug=org_slug
+            )
+        if not election_model.organization:
+            election_model.organization = organization_extra.base
+            election_model.save()
+        return organization_extra
+
+    def add_area(self, area_info, election, organization_extra):
+        if area_info['organisation']['territory_code'] == "NIR":
+            partyset_name = "Northern Ireland"
+            country = "ni"
+        else:
+            partyset_name = "Great Britain"
+            country = "gb"
+
+        party_set, _ = PartySet.objects.get_or_create(
+            slug=country, defaults={'name': partyset_name}
+        )
+
+        if area_info['division']:
```

Comment:
I think that currently there's a strong assumption for the UK that the `name` of an `AreaType` corresponds to a MapIt `Type` - I just wanted to check that that's still the case (or it might be worth a comment in the code to that effect).